lib/config/presets/internal/workarounds.ts

```diff
@@ -13,6 +13,7 @@ export const presets: Record<string, Preset> = {
       'workarounds:typesNodeVersioning',
       'workarounds:reduceRepologyServerLoad',
       'workarounds:doNotUpgradeFromAlpineStableToEdge',
+      'workarounds:ignoreDotNet7',
```

Comment:
```suggestion
      'workarounds:ignoreDotnet7Preview',
```

Reply:
I think this will enable it for almost all, as it's in config:base. So this line should be removed then.